app.js

```diff
@@ -19,8 +19,26 @@ app.use(function(req, res, next) { // enable CORS and assume JSON return structu
   next();
 });
 var hash = new hashid(process.env.HASH_SALT);
-var db = new loki(__dirname+'/main.json'); // intiialize datastore
-var pending_requests = db.addCollection("pending_requests"); // pending requests
+
+var connection = null;
```

Comment:
That's how Rethink's documentation explains it. 

Basically the advantage to storing the connection from Rethink in a global connection variable is so you don't need to reinitialize the database connection each time you want to query it.